README.md

```diff
@@ -56,6 +56,8 @@ assert all() == :mocked
 refute filtered() == :mocked
 ```
 
+**Note**: Module names are passed as a string (`"MyApp.UserService`") instead o atoms (`MyApp.UserService`). This reduces the compilation time because it doesn't create a link between modules which caused modules to be recompiled too often. This doesn't affect the bahaviour in any way.
+
```

Comment:
Add second paragraph about erlang modules like `:crypto`, they should be used as atoms